sql/parse/parse_test.go

```diff
@@ -26,849 +26,793 @@ var showCollationProjection = plan.NewProject([]sql.Expression{
-		&sql.ForeignKeyConstraint{
-			Name:              "",
-			Columns:           []string{"b_id"},
-			ReferencedTable:   "t0",
-			ReferencedColumns: []string{"b"},
-			OnUpdate:          sql.ForeignKeyReferenceOption_DefaultAction,
-			OnDelete:          sql.ForeignKeyReferenceOption_Restrict,
-		},
-	),
-	`ALTER TABLE t1 ADD FOREIGN KEY (b_id) REFERENCES t0(b) ON UPDATE SET NULL ON DELETE NO ACTION`: plan.NewAlterAddForeignKey(
-		plan.NewUnresolvedTable("t1", ""),
-		plan.NewUnresolvedTable("t0", ""),
-		&sql.ForeignKeyConstraint{
-			Name:              "",
-			Columns:           []string{"b_id"},
-			ReferencedTable:   "t0",
-			ReferencedColumns: []string{"b"},
-			OnUpdate:          sql.ForeignKeyReferenceOption_SetNull,
-			OnDelete:          sql.ForeignKeyReferenceOption_NoAction,
-		},
-	),
-	`ALTER TABLE t1 ADD FOREIGN KEY (b_id, c_id) REFERENCES t0(b, c)`: plan.NewAlterAddForeignKey(
-		plan.NewUnresolvedTable("t1", ""),
-		plan.NewUnresolvedTable("t0", ""),
-		&sql.ForeignKeyConstraint{
-			Name:              "",
-			Columns:           []string{"b_id", "c_id"},
-			ReferencedTable:   "t0",
-			ReferencedColumns: []string{"b", "c"},
-			OnUpdate:          sql.ForeignKeyReferenceOption_DefaultAction,
-			OnDelete:          sql.ForeignKeyReferenceOption_DefaultAction,
-		},
-	),
-	`ALTER TABLE t1 ADD CONSTRAINT fk_name FOREIGN KEY (b_id, c_id) REFERENCES t0(b, c) ON UPDATE RESTRICT ON DELETE CASCADE`: plan.NewAlterAddForeignKey(
-		plan.NewUnresolvedTable("t1", ""),
-		plan.NewUnresolvedTable("t0", ""),
-		&sql.ForeignKeyConstraint{
-			Name:              "fk_name",
-			Columns:           []string{"b_id", "c_id"},
-			ReferencedTable:   "t0",
-			ReferencedColumns: []string{"b", "c"},
-			OnUpdate:          sql.ForeignKeyReferenceOption_Restrict,
-			OnDelete:          sql.ForeignKeyReferenceOption_Cascade,
-		},
-	),
-	`ALTER TABLE t1 DROP FOREIGN KEY fk_name`: plan.NewAlterDropForeignKey(
-		plan.NewUnresolvedTable("t1", ""),
-		&sql.ForeignKeyConstraint{
-			Name:              "fk_name",
-			Columns:           []string{},
-			ReferencedTable:   "",
-			ReferencedColumns: []string{},
-			OnUpdate:          sql.ForeignKeyReferenceOption_DefaultAction,
-			OnDelete:          sql.ForeignKeyReferenceOption_DefaultAction,
-		},
-	),
-	`ALTER TABLE t1 DROP CONSTRAINT fk_name`: plan.NewAlterDropForeignKey(
-		plan.NewUnresolvedTable("t1", ""),
-		&sql.ForeignKeyConstraint{
-			Name: "fk_name",
-		},
-	),
-	`DESCRIBE foo;`: plan.NewShowColumns(false,
-		plan.NewUnresolvedTable("foo", ""),
-	),
-	`DESC foo;`: plan.NewShowColumns(false,
-		plan.NewUnresolvedTable("foo", ""),
-	),
-	"DESCRIBE FORMAT=tree SELECT * FROM foo": plan.NewDescribeQuery(
-		"tree", plan.NewProject(
-			[]sql.Expression{expression.NewStar()},
-			plan.NewUnresolvedTable("foo", ""),
-		)),
-	"DESC FORMAT=tree SELECT * FROM foo": plan.NewDescribeQuery(
-		"tree", plan.NewProject(
-			[]sql.Expression{expression.NewStar()},
+func init() {
```

Comment:
This change makes the diff impossible to review. Can you summarize why this can no longer be statically initialized?

Reply:
This is a relic from when I was using `init()` functions everywhere. This has been fixed.